aten/src/ATen/native/native_functions.yaml

```diff
@@ -3497,22 +3497,22 @@
     CPU: gelu_out_cpu
     CUDA: gelu_out_cuda
 
-- func: gelu(Tensor self) -> Tensor
+- func: gelu(Tensor self, bool approximate) -> Tensor
```

Comment:
nitpick, should we default approximate to False just so that we don't have to update tests?